src/main/java/frc/robot/subsystems/trapElvSubsystem/TrapElvCommandFactory.java

```diff
@@ -0,0 +1,179 @@
+package frc.robot.subsystems.trapElvSubsystem;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.StartEndCommand;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.Constants;
+import frc.robot.Constants.TrapElvConstants;
+import frc.robot.subsystems.trapElvSubsystem.TrapElvSubsystem.TrapElvState;
+import java.util.function.BooleanSupplier;
+
+public class TrapElvCommandFactory {
+  private final TrapElvSubsystem subsystem;
+
+  public TrapElvCommandFactory(TrapElvSubsystem subsystem) {
+    this.subsystem = subsystem;
+  }
+
+  public Command stowTrapElvCommand() {
+    if (subsystem == null) return new InstantCommand();
```

Comment:
Nitpick/Optional: Commands.none() does the same thing but makes the intent a bit more obvious.
```suggestion
    if (subsystem == null) return Commands.none();
```